### ISSUE TITLE
RChain.coop GDPR compliance

### ISSUE THREAD

**Issue by JoshOrndorff** [rchain/marketing#10]
rchain.coop needs to be GPDR compliant and inform European users about cookie usage.

Opened by @kitblake in the old trello

**Comment by bjornagain** [comment]
The site doesn't currently collect any data, outside of the email address for newsletter signup, and we aren't using any cookies. What should the site say about cookie usage?

I actually just noticed that we don't have links set up in the footer for 'privacy', 'terms and conditions'. Does anyone one have those links?

**Comment by JoshOrndorff** [comment]
That's excellent news and will be attractive to the libertarians in the crypto space. We should let people know we don't use cookies or spy on them. I've always liked the JAXX privacy policy which starts with "privacy is a human right ans software should help protect your rights".

On the other hand does the blog count as part of the website? It certainly would if we do #4 )I believe they do use google analytics, right @plantether ?

**Comment by JoshOrndorff** [comment]
@KWattage Can you draft something up for privacy policy? Do we really need terms and conditions? I vote no.

**Comment by plantether** [comment]
@JoshOrndorff That's true, We do use cookies and spy on people using GA. There is an action item to add the notice to the blog.  The blog will soon live on the main rchain.coop site. @KWattage are there plans to use GA for rchain.coop in general?
